sdk/iothub/arm-iothub/src/models/index.ts

```diff
@@ -134,23 +134,15 @@ export interface IotHubProperties {
   cloudToDevice?: CloudToDeviceProperties;
   /** IoT hub comments. */
   comments?: string;
-  /** The device streams properties of iothub. */
-  deviceStreams?: IotHubPropertiesDeviceStreams;
```

Comment:
Is this not a breaking because it's added in the preview but not exist in hte latest stable?

Reply:
yes